src/microsAuthentication/content.md

```diff
@@ -0,0 +1,1129 @@
+kubectl apply -f sa.yaml
+namespace/test created
+serviceaccount/sa-test-1 created
+```
+
+To view the token associated with the service account:
+
+```terminal|command=1|title=bash
+kubectl --namespace test describe sa sa-test-1
+Name:                 sa-test-1
+Namespace:            test
+Labels:               <none>
+Image pull secrets:   <none>
+Mountable secrets:    sa-test-1-token-99l5r
+Tokens:               sa-test-1-token-99l5r
+Events:               <none>
+```
+
+Then to view the secret object, you can issue the following command:
+
+```terminal|command=1|title=bash
+kubectl --namespace test describe secret sa-test-1-token-99l5r
+Name:       sa-test-1-token-99l5r
+Namespace:  test
+Labels:     <none>
+Type:  kubernetes.io/service-account-token
+
+Data
+====
+ca.crt:     1066 bytes
+namespace:  4 bytes
+token:      eyJhbGciOiJSUzI1NiIsImtpZCI6…
+```
+
+The `token` object in the Data is a base64 encoded object representing a JSON web token payload.
+
+Once you have an appropriate RoleBinding or ClusterRoleBinding setup, you can use this token make requests to the Kubernetes API.
+
+For example, you might want to verify whether the token is valid or not by invoking the Kubernetes TokenReview API.
+
+### Verifying a token using TokenReview API
+
+A Service Account is just an identity with no permissions associated with it.
+
+If you want to grant access to the Service account you need two more objects: a Binding and a Role.
+
+In the Role you define access to the resources such as read-only access to Secrets or only GET, DESCRIBE and LIST operations on Pods.
+
+Kubernetes comes with a list of roles prepackaged that you can list with:
+
+```terminal|command=1|title=bash
+kubectl get roles --all-namespaces
+
+NAMESPACE     NAME                                             CREATED AT
+kube-public   kubeadm:bootstrap-signer-clusterinfo             2020-08-01T08:07:34Z
+kube-public   system:controller:bootstrap-signer               2020-08-01T08:07:33Z
+kube-system   extension-apiserver-authentication-reader        2020-08-01T08:07:32Z
+kube-system   kube-proxy                                       2020-08-01T08:07:35Z
+kube-system   kubeadm:kubelet-config-1.18                      2020-08-01T08:07:33Z
+kube-system   kubeadm:nodes-kubeadm-config                     2020-08-01T08:07:33Z
+kube-system   system::leader-locking-kube-controller-manager   2020-08-01T08:07:33Z
+kube-system   system::leader-locking-kube-scheduler            2020-08-01T08:07:33Z
+kube-system   system:controller:bootstrap-signer               2020-08-01T08:07:32Z
+kube-system   system:controller:cloud-provider                 2020-08-01T08:07:32Z
+kube-system   system:controller:token-cleaner                  2020-08-01T08:07:33Z
+kube-system   system:persistent-volume-provisioner             2020-11-20T22:04:53Z
+```
+
+In the following example, you will use the `system:auth-delegator` role which has all the permission you need to call the Token Review API.
```

Comment:
There's no `system:auth-delegator` when I list the roles. Not even in the table above. How did you create that?